pyclient/pymtt.py

```diff
@@ -158,7 +155,8 @@
 # any set on the command line.  Environment will override the commandline.
 mttArgs = []
 if 'MTT_ARGS' in os.environ and os.environ.get('MTT_ARGS') != None:
-    mttArgs = os.environ['MTT_ARGS'].split()
+    # find all the ' --' command line args and split them
+    mttArgs = os.environ['MTT_ARGS'].replace(' --', ';--').split(';')
```

Comment:
Why would we do this? I'm not sure I understand the rationale.

Reply:
here is an example, with MTT_ARGS set like
  `MTT_ARGS='--description=this is a test --skip-sections=Reporter:IUDatabase'`
we cannot split on space, but instead we need to split on something like ' --' to get each argument, but splitting 'eats' the characters you split on, so I change the ' --' to ';--' then split on ';' ...

Reply:
maybe if you split using shlex?

`>>> shlex.split("--description='this is a test' --skip-sections=Reporter:IUDatabase")
['--description=this is a test', '--skip-sections=Reporter:IUDatabase']
`

Reply:
ok, I like this, it supports quoting a string with spaces, like the description which is more natural